autotest/gdrivers/ngw.py

```diff
@@ -69,38 +72,56 @@ def startup_and_cleanup():
 
 
 def check_availability(url):
-    # Sandbox cleans at 1:05 on monday (UTC)
-    now = datetime.utcnow()
-    if now.weekday() == 0:
-        if now.hour >= 0 and now.hour < 4:
-            return False
 
     version_url = url + "/api/component/pyramid/pkg_version"
 
-    if gdaltest.gdalurlopen(version_url) is None:
+    if gdaltest.gdalurlopen(version_url, timeout=NET_TIMEOUT) is None:
         return False
 
-    # Check quota
-    quota_url = url + "/api/resource/quota"
-    quota_conn = gdaltest.gdalurlopen(quota_url)
-    try:
-        quota_json = json.loads(quota_conn.read())
-        quota_conn.close()
-        if quota_json is None:
-            return False
-        limit = quota_json["limit"]
-        count = quota_json["count"]
-        if limit is None or count is None:
-            return True
-        return limit - count > 15
-    except Exception:
-        return False
+    return True
 
 
 def get_new_name():
     return "gdaltest_group_" + str(int(time.time())) + "_" + str(random.randint(10, 99))
 
 
+def check_tms(sub):
+    ds = gdal.OpenEx(
+        sub[0],
+        gdal.OF_RASTER,
+        open_options=[
+            f"TIMEOUT={NET_TIMEOUT}",
+            f"MAX_RETRY={NET_MAX_RETRY}",
+            f"RETRY_DELAY={NET_RETRY_DELAY}",
+        ],
+    )
+
+    assert ds is not None, f"Open {sub[0]} failed."
+
+    assert (
+        ds.RasterXSize == 1073741824
+        and ds.RasterYSize == 1073741824
+        and ds.RasterCount == 4
+    ), f"Wrong size or band count for raster {sub[0]} [{sub[1]}]."
+
+    wkt = ds.GetProjectionRef()
+    assert 'PROJCS["WGS 84 / Pseudo-Mercator"' in wkt, "Got wrong SRS: " + wkt
+
+    gt = ds.GetGeoTransform()
+    # -20037508.34, 0.037322767712175846, 0.0, 20037508.34, 0.0, -0.037322767712175846
+    assert (
```

Comment:
you can use assert gt == pytest.approx(expected_gt, abs=0.00001) to test the whole gt at once